src/wp-includes/blocks/query-title.php

```diff
@@ -28,7 +28,7 @@ function render_block_core_query_title( $attributes ) {
 	if ( $is_archive ) {
 		$show_prefix = isset( $attributes['showPrefix'] ) ? $attributes['showPrefix'] : true;
 		if ( ! $show_prefix ) {
-			$filter_title = function( $title, $original_title ) {
+			$filter_title = static function( $title, $original_title ) {
```

Comment:
Note: This code doesn't exist in Gutenberg. It instead has:

https://github.com/WordPress/gutenberg/blob/e3194a1d1d9869560260a3bba0939a28a52e737f/packages/block-library/src/query-title/index.php#L31-L33